lamindb/models/artifact.py

```diff
@@ -2706,6 +2709,9 @@ def _track_run_input(
     is_run_input: bool | Run | None = None,
     run: Run | None = None,
 ):
+    if is_run_input is False:
+        return
```

Comment:
@falexwolf I think it should just return if `False` instead of pulling `run` etc. Am i correct?

Reply:
I think so, yes!